src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapper.java

```diff
@@ -416,6 +421,32 @@ private KNNEngine validateDimensions(final KNNMethodContext knnMethodContext, fi
             }
             return knnEngine;
         }
+
+        /**
+         * Validate whether provided full field name contain any invalid characters for physical file name.
+         * At the moment, we use a field name as a part of file name while we throw an exception
+         * if a physical file name contains any invalid characters when creating snapshot.
+         * To prevent from this happening, we restrict vector field name and make sure generated file to have a valid name.
+         *
+         * @param context : Builder context to have field name info.
+         */
+        private void validateFullFieldName(BuilderContext context) {
```

Comment:
```suggestion
        private void validateFullFieldName(final BuilderContext context) {
```

Reply:
Thank you for the suggestion, but I don't think I would add `final` keyword unless you have strong opinion on this.
Unlike `const` keyword in C++, I hardly think introducing `final` adds any values other than prohibiting reassignment.
As you can see in the method, it just merely bypasses the reference, and it would not make any differences whether if we added `final`.

Of course, please feel free drop any thoughts on this! :)

Reply:
It is just coding convention we are following. KNN does not have published document about it though. We might need one..

Reply:
Oh.. if that is the case then I will update accordingly!

Reply:
Updated!